docs/source/datasets/index.rst

```diff
@@ -121,59 +128,75 @@ The following columns are included in this dataset:
      - Notes
    * - Unique simulant ID
      - :code:`simulant_id`
-     - Not affected by noise functions; intended use is "ground truth" for testing and validation. 
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
    * - Unique household ID
      - :code:`household_id`
      - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
        datasets.
    * - First name
      - :code:`first_name`
-     - 
+     -
    * - Middle initial
      - :code:`middle_initial`
-     - 
+     -
    * - Last name
      - :code:`last_name`
-     - 
+     -
    * - Age
-     - :code:`age`  
+     - :code:`age`
      - Rounded down to an integer.
    * - Date of birth
      - :code:`date_of_birth`
      - Formatted as YYYY-MM-DD.
    * - Physical address street number
      - :code:`street_number`
-     - 
+     -
    * - Physical address street name
      - :code:`street_name`
-     - 
+     -
    * - Physical address unit number
      - :code:`unit_number`
-     - 
+     -
    * - Physical address city
-     - :code:`city`    
-     - 
+     - :code:`city`
+     -
    * - Physical address state
-     - :code:`state`  
-     - 
+     - :code:`state`
+     -
    * - Physical address ZIP code
      - :code:`zipcode`
-     - 
-   * - Sex 
-     - :code:`sex`  
+     -
+   * - Housing type
+     - :code:`housing_type`
+     - Possible values for housing type are "Household" for an individual
+       household, or one of six different types of group quarters. The types of
+       instiutional group quarters are "Carceral", "Nursing home", and "Other
+       institutional". The types of noninstitutional group quarters are
+       "College", "Military", and "Other noninstitutional".
+   * - Relationship to reference person
+     - :code:`relationship_to_reference_person`
+     - Possible values for this indicator include:
+       "Reference person"; "Opposite-sex spouse"; "Opposite-sex unmarried
+       partner"; "Same-sex spouse"; "Same-sex unmarried partner"; "Biological
+       child"; "Adopted child"; "Stepchild"; "Sibling"; "Parent"; "Grandchild";
+       "Parent-in-law"; "Child-in-law"; "Other relative"; "Roommate"; "Foster
+       child"; "Other nonrelative"; "Institutionalized group quarters
+       population"; and "Noninstitutionalized group quarters population".
```

Comment:
I used the full category names such as "Opposite-sex unmarried partner" and "Noninstitutionalized group quarters population" instead of our abbreviations such as "Opp-sex partner" and "Noninstitutionalized GQ pop". I think that's better -- do other people have a preference?

Reply:
agree, since the strings are long anyway we might as well have them unabbreviated.

Reply:
If we're changing the "GQ pop" ones anyway, did we decide whether we like the word "population"? I feel like "group quarters person" is more natural for this column but don't feel strongly.